components/audio/PirateAudioHAT/setup_pirateAudioHAT.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+question() {
+    local question=$1
```

Comment:
Would make sense to have a "library" somewhere for generic script functions like that. At least as future improvement.
Sorry I'm taking advantage of you as you are capable of writing high quality bash scripts. 😉

Reply:
I like libraries in general. Right now, it does not matter where this script is executed and it does not depend on any other script, which is quite handy. Once we repeat stuff too many times, we should definitely group it somewhere though.

Reply:
Agreed.

Reply:
Agreed.